hedera-node/docs/system-accounts-operations.md

```diff
@@ -0,0 +1,187 @@
+| 0xA → 0x166 (0.0.10 → 0.0.358)     | ✅ - if address is a contract and we’re using the correct ABI<br/>✅ - success with no op, if there is no contract<br/>❌ - if address is a contract and we’re not using the correct ABI | ✅ - success with no op                                                                                       |
+| 0x167 (0.0.359)                    | ✅ - if address is a contract and we’re using the correct ABI<br/>✅ - success with no op, if there is no contract<br/>❌ - if address is a contract and we’re not using the correct ABI | ✅ - HTS system contract                                                                                      |
+| 0x168, 0x169 (0.0.360, 0.0.361)    | ✅ - if address is a contract and we’re using the correct ABI<br/>✅ - success with no op, if there is no contract<br/>❌ - if address is a contract and we’re not using the correct ABI | ✅ - ExchangeRate and PRNG system contracts                                                                   |
+| 0x16a → 0x2EE (0.0.362 → 0.0.750)  | ✅ - if address is a contract and we’re using the correct ABI<br/>✅ - success with no op, if there is no contract<br/>❌ - if address is a contract and we’re not using the correct ABI | ✅ - success with no op                                                                                       |
+| 0x2EF → 0x3E8 (0.0.751 → 0.0.1000) | ✅ - if address is a contract and we’re using the correct ABI<br/>✅ - success with no op, if there is no contract<br/>❌ - if address is a contract and we’re not using the correct ABI | ✅ - success with no op                                                                                       |
+| 0x3E8 → ♾️ | ✅ - if address is a contract and we’re using the correct ABI<br/>✅ - success with no op, if there is no contract                                                                 | ✅ - if a contract exists and we’re using the correct ABI<br/>✅ - success with no op, if there is no contract |
+
+### Transfer and send operations
+
+**Ethereum:**
+1. For addresses in range 0x0 → 0x9: success, value is locked.
+2. For addresses 0xA and above: 
+    - success, if address is a payable contract;
+    - success, if address is an account;
+    - success, if address is empty + creates the account;
+    - fail, if address is a non-payable contract.
+
+**Hedera:**
+1. For addresses in range 0x0 → 0x2EE (0.0.0 → 0.0.750):
+    - Transfers using `.send` and `.transfer`: fail with status `INVALID_FEE_SUBMITTED`. Exception: `tokenCreate` to address 0x167 which is successful.
+    - Transfer of HTS tokens through the HTS system contract: fail with status `INVALID_RECEIVING_NODE_ACCOUNT`.
+2. For addresses in range 0x2EF → 0x3E8 (0.0.751 → 0.0.1000):
+    - success, if account exists and has `receiverSigRequired` == false;
+    - success, if account exists and has `receiverSigRequired` == false and account is `sender`;
+   
+_The account is not created if it doesn't exist._
+3. For addresses above 0.0.1000:
```

Comment:
The formatting of point 3 is wrong. It doesn't display as a table but plain text.

Reply:
This line is not meant to be a part of a table. The table starts from line 178.